Makefile.defines

```diff
@@ -111,8 +111,14 @@ LDFLAGS  += -nostartfiles
 endif
 
 ifeq ($(TARGET_NAME),TARGET_NANOS)
-DEFINES += HAVE_BAGL
 DEFINES += BAGL_WIDTH=128 BAGL_HEIGHT=32
+
+ifeq ($(USE_NBGL),0)
```

Comment:
should we cover if USE_NBGL is undefined ?

Reply:
ifneq ($(USE_NBGL),1)
?

Reply:
I have aligned on Ledger-secure-sdk master branch version